website/project/model.py

```diff
@@ -118,6 +120,31 @@ def requires_consent(self):
     def has_files(self):
         return self._config.get('hasFiles', False)
 
+    @property
+    def absolute_api_v2_url(self):
+        path = '/metaschemas/{}/'.format(self._id)
+        return api_v2_url(path)
+
+    # used by django and DRF
+    def get_absolute_url(self):
+        return self.absolute_api_v2_url
+
+    def validate_metadata(self, *args, **kwargs):
```

Comment:
Would be good to have unit tests for this method, but those can be added in a follow up PR.